dependent-mapping/README.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: pattern
+title: Dependent Mapping
+folder: dependent-mapping
+permalink: /patterns/dependent-mapping/
+categories: Behavioral
+language: en
+tags:
+ - Database
+ - consistency
+---
```

Comment:
Study https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute to see how the yaml frontmatter should be implemented

I recommend using category `Structural` and tags `Data access`. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags